src/Umbraco.Core/Configuration/Models/SecuritySettings.cs

```diff
@@ -26,6 +26,7 @@ public class SecuritySettings
     internal const int StaticMemberDefaultLockoutTimeInMinutes = 30 * 24 * 60;
     internal const int StaticUserDefaultLockoutTimeInMinutes = 30 * 24 * 60;
     internal const string StaticAuthorizeCallbackPathName = "/umbraco";
+    internal const string StaticAuthorizeCallbackErrorPathName = StaticAuthorizeCallbackPathName + "/error";
```

Comment:
It's not entirely safe to concatenate these strings, as we might want to change the authorize callback pathname to something like "/umbraco/oauth_complete" in the future, and so, you would end up with "/umbraco/oauth_complete/error", so best to code the entire string into the const here:

```suggestion
    internal const string StaticAuthorizeCallbackErrorPathName = "/umbraco/error";
```